package.json

```diff
@@ -122,7 +123,7 @@
     "ignore": [
       "**/*.test.ts"
     ],
-    "exec": "npm run build && source ./dist/config/.env && node dist/server.js"
+    "exec": "node --inspect=0 -r ts-node/register src/server.ts"
```

Comment:
will this have an issue on dev and live envs?

Reply:
Hey - I don't think so, this is for running with `nodemon` for local development

This change allows us to use a debugger on the TypeScript rather than the transpiled JavaScript